ambry-store/src/main/java/com/github/ambry/store/ReplicationProtocolTransitionType.java

```diff
@@ -0,0 +1,73 @@
+package com.github.ambry.store;
+
+public enum ReplicationProtocolTransitionType {
```

Comment:
[Info] Proposing this way to thoroughly test BootstrapController's logic.
Tests are making use of these enum values and asserting if the logic takes an unexpected path.